x-pack/packages/kbn-elastic-assistant/impl/assistant/index.tsx

```diff
@@ -446,44 +447,58 @@ const AssistantComponent: React.FC<Props> = ({
             {Object.keys(promptContexts).length > 0 && <EuiSpacer size={'s'} />}
           </>
         )}
+      </EuiModalHeader>
+      <EuiModalBody>
```

Comment:
By putting the dialog in `EuiModalBody`, we allow eui to handle the scrolling behavior. The `max-height` in now deleted `CommentsContainer` was causing the modal to expand unexpectedly

Reply:
really nice!